js/script.js

```
@@ -212,6 +221,185 @@ class FruitWebsite {
+        if (!sortSelect) return;
+        
+        sortSelect.addEventListener('change', () => {
+            const sortValue = sortSelect.value;
+            const fruitsGrid = document.querySelector('.fruits-grid');
+            const fruitCards = Array.from(fruitsGrid.querySelectorAll('.fruit-card'));
+            
+            // Sort cards
+            fruitCards.sort((a, b) => {
+                switch (sortValue) {
+                    case 'name':
+                        const nameA = a.querySelector('.fruit-name').textContent;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)